frontend/catalyst/jax_tracer.py

```diff
@@ -486,12 +489,18 @@ def _bind_native_controlled_op(qrp, op, controlled_wires, controlled_values):
         if isinstance(op, HybridOp) and not isinstance(
             op, catalyst.api_extensions.quantum_operators.HybridCtrl
         ):
-            qrp2 = op.trace_quantum(ctx, device, trace, qrp)
+            kwargs = (
+                {
+                    "postselect_mode": getattr(qnode, "execute_kwargs", {})
+                    .get("mcm_config", qml.devices.MCMConfig())
+                    .postselect_mode
+                }
+                if isinstance(op, catalyst.api_extensions.quantum_operators.MidCircuitMeasure)
+                else {}
+            )
+            qrp2 = op.trace_quantum(ctx, device, trace, qrp, **kwargs)
```

Comment:
Would it be simpler to just pass in the execution or mcm config? Fairly minor admittedly but it could make the signature more uniform.

Reply:
I updated it to take `mcm_config` as an optional kwarg.